_docker/drupal/drupal-filesystem/web/themes/custom/rhd/rhd.theme

```diff
@@ -103,12 +109,12 @@ function rhd_js_settings_alter(array &$settings, \Drupal\Core\Asset\AttachedAsse
 
     $rhd_settings['downloadManager'] = array();
     $rhd_settings['downloadManager']['baseUrl'] = $env_settings->get('downloadManager')['baseUrl'];
-    $rhd_settings['dcp']['baseProtocolRelativeUrl'] = $env_settings->get('searchisko')['protocol'] . "://" . $env_settings->get('searchisko')['host'] . ":"
```

Comment:
This needs to be reverted back to the `$env_settings->get('searchisko')['protocol']` otherwise we will break the functionality of the site over https by loading from a http source.